lib/src/model/mi_scale_extra_data.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:xiaomi_scale/src/model/gender.dart';
+
+class MiScaleExtraData {
```

Comment:
I feel like this might be slightly more descriptive about what it is.
```suggestion
class MiScaleBodyData {
```

Reply:
I will update that